src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -278,11 +278,13 @@ const AOVContainer& Frame::internal_aovs() const
     return impl->m_internal_aovs;
 }
 
-size_t Frame::create_extra_aov_image(const char* name) const
+size_t Frame::create_extra_aov_image(
+    const char* name,
+    const PixelFormat pixel_format) const
```

Comment:
Personally, I prefer to undo this change and remove the option of saving extra aov images to pngs.
The images in the extra aov images are always non-color images and in my opinion, it doesn't make much sense to save them as pngs.